pkgs/development/python-modules/pysubs2/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-cov
+, mypy
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "pysubs2";
+  version = "1.6.1";
+  format = "pyproject";
+
+  # pypi is missing the 'tests/data' directory, needed to run tests
+  src = fetchFromGitHub {
+    owner = "tkarabela";
+    repo = pname;
+    rev =  version;
+    hash = "sha256-0bW9aB6ERRQK3psqeU0Siyi/8drEGisAp8UtTfOKlp0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov
+    mypy
```

Comment:
mypy is a linter, is it needed to test?

Reply:
even if it is needed, we should remove it